patternfly/patternfly-elements
# fix: set margin-top for ctas in footer on mobile

## Description
Margin-top is not applied to the CTAs in pfe-navigation footer after the fourth. Set the Light DOM styles for mobile so that the fifth (and subsequent) CTAs have the correct margins.

### Preview


Link(s) to demo page(s) where this element can be viewed:
- [Link](https://deploy-preview-1667--patternfly-elements.netlify.app/components/navigation/demo/) 


### What has changed and why

-  In `pfe-navigation--lightdom.scss` at breakpoint-sm, changed the selector containing the margin-top property from `nth-child(3), nth-child(4)` to `nth-child(n+3)`


### Testing instructions



  - Visit https://deploy-preview-1667--patternfly-elements.netlify.app/elements/pfe-navigation/demo/index.html
  - Open up the browser console and paste in the following code.
  ```
  let template = `<div class="pfe-navigation--column"><pfe-cta><a href="#">Start a trial</a></pfe-cta></div>`;
  document.querySelector('pfe-navigation-main [slot="tray"] .pfe-navigation--footer').insertAdjacentHTML('beforeend', template);
  ```
  - Resize the browser to `900px`
  - Verify that the top margin of the 2nd, 3rd, and 4th are the same
  
  
![Screen Shot 2021-06-07 at 4 19 46 PM](https://user-images.githubusercontent.com/3428964/121082599-52a85d00-c7ac-11eb-91b1-7e36d1090ea2.png)



#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist





### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
